vnext/local-cli/tsconfig.json

```diff
@@ -12,5 +12,5 @@
     // Not clean
     "strictFunctionTypes": false,
   },
-  "include": ["src"],
+  "include": ["src", "src/generator-windows/index.ts"],
```

Comment:
I'm not sure this change does anything. Artifact from an older iteration?

Reply:
Missed reverting this file, something has been applying funny refactorings